RELEASING.md

```diff
@@ -110,6 +110,16 @@ OpenSearch only does major releases when there are significant breaking changes
 
 For the actual steps to build a release, please see [Releasing OpenSearch](https://github.com/opensearch-project/opensearch-build/blob/main/README.md#releasing-opensearch).
 
+### Release Process for Changing Release Date
```

Comment:
Add to TOC please.

How about we make a section called "Release Process" and sub-topics "Major Releases" and "Changing Release Dates"?

Reply:
I changed releasing to "Release Process" and included everything underneath it. Simplified the whole section.